File: python/ejer25.py
#Mostrar menú:
nombre = input("Ingrese su nombre:")

bienvenida ="Bienvenido al sistema " + nombre 
print(bienvenida)

#Mostrar opciones:
print("1. cliente normal")    
print("2. cliente vip")

opcion = int(input("Ingrese su opción:"))

if opcion == 1:
    print("Usted es un cliente normal, su descuento es del 5%")

elif opcion == 2:
    print("Usted es un cliente vip, su descuento es del 20%")

File: js/ejer25.js
//Mostrar menú:
nombre = prompt("Ingrese su nombre:");
menu="bienvenido al sistema " + nombre;
fecha = new Date();
menu += "\nFecha: " + fecha.toLocaleDateString();
menu += "\nHora: " + fecha.toLocaleTimeString();

alert(menu)


cliente = prompt("Seleccione el tipo de cliente: \n1. Cliente Regular \n2. Cliente VIP");

switch (cliente) {
    case "1":
        alert("Has seleccionado Cliente Regular tienes un descuento del 5%");

}

        switch (cliente) {
    case "2":
        alert("Has seleccionado Cliente VIP tienes un descuento del 20%");}



